README.md

```diff
@@ -39,3 +39,29 @@ The plugin can be debugged in the developer tools of Chrome. If you have a local
 add a line with the `debugger` command in the JavaScript code to stop at this location, because it is sometimes
 difficult to identify the file between the several hundred (!) files that Outlook loads.
 
+# Customer Documentation
+
+## Install Outlook plugin
+To install the Outlook Plugin go to the browser version of outlook and deactivate the new outlook theme:
+
+![Old Outlook Theme](/documentation/oldOutlook.png)
+
+Next click on `Manage add-ins`:
+
+![Manage Outlook](/documentation/manageAddons.png)
+
+Find `upload from url` from the `My add-ins` section and insert the manifest.xml url `https://assistify.github.io/outlook-plugin/manifest.xml`:
+
+![Upload From Url](/documentation/uploadFromUrl.png)
+
+After you installed the Plugin a new icon should appear in your Email window
```

Comment:
Change `Email window` to Outlook for Windows / Mac / Desktop and add a screenshot which shows the integration